lib/routes/default.rb

```diff
@@ -12,6 +12,10 @@ def self.registered(puppet_webhook)
         puppet_webhook.get '/heartbeat' do
           return 200, { status: :success, message: 'running' }.to_json
         end
+
+        puppet_webhook.post '/echo' do
```

Comment:
I'm not sure that we should have this here. The function that this endpoint is providing can already be found via Ruby development tools, the `puppet_webhook` log, and the spec tests.